keanu-python/tests/test_vertex.py

```diff
@@ -175,3 +186,34 @@ def test_get_vertex_id(jvm_view):
     python_id = gaussian.get_id()
 
     assert all(value in python_id for value in java_id)
+
+@pytest.mark.parametrize("mu, sigma", [
```

Comment:
You can actually parametrise `mu` and `sigma` independently, using 2 `@parametrize` blocks.

This kind of thing:

    @pytest.mark.parametrize("mu", [
        0, False, np.array([0]), np.array([False])
    ])
    @pytest.mark.parametrize("sigma", [
        1, True, np.array([1]), np.array([True])
    ])

Reply:
This is amazing! I think this stacking tests all combinations of `mu` and `sigma`, right?